code/sorting/sorting.py

```diff
@@ -0,0 +1,32 @@
+stringType = type('')
+
+
+class Sorting:
+    def __init__(self, array):
+        self.array = array
+
+    def convert_to_array_of_int(self):
+        array_of_int = []
+
+        for i in self.array:
+            if type(i) == stringType and not i.isalpha():
+                array_of_int.append(int(i))
+            if type(i) == stringType and not i.isalpha():
+                continue
+            if type(i) != stringType:
+                array_of_int.append(i)
```

Comment:
unreasonable duplication of "if" branches and wrong logic (.isalpha() return false for string "123abc")

```suggestion
            if isinstance(i, str) and i.isnumeric():
                array_of_int.append(int(i))
```

Reply:
done